CONTRIBUTING.md

```diff
@@ -224,9 +224,19 @@ The design review process is as follows:
   if (GVFSPlatform.Instance.KernelDriver.EnumerationExpandsDirectories)
   ```
 
-- *Add new interfaces when it makes sense for the product, not simply for unit testing*
+- *Testability of a class should be a priority in the design of a class and during code reviews.*
```

Comment:
📎 says:
Consider revising this sentence to avoid specifying the subject twice:
> - *Testability ~of a class~ should be a priority in the design of a class and during code reviews.*